assets/css/style1.css

```diff
@@ -14,25 +14,33 @@ body {
 }
 
 .stickyFooter {
-	flex: 1 0 auto;
+	min-height: 100%;
```

Comment:
These should be applied to the body tag I think, not the footer:
```
min-height: 100%;
margin-bottom: -50px;
```
 `margin-bottom` should be offset by whatever height you set the footer to (ie. if footer height is 50px, `margin-bottom` for body should be -50px):

The footer itself should just have a `height` property (ie. `height: 50px`)